File: Domain/Sector.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain
{
    public class Sector
    {
        public Guid Id { get; set; }
        public string Description { get; set; }
        public bool IsDeleted { get; set; } = false;
        public ICollection<ServiceOwner> ServiceOwners { get; set; }
        public ICollection<Service> Services { get; set; }
        public ICollection<CategoryService> CategoryServices { get; set; }



    }
}

File: ScheduledServicesCenterUI/src/app/sector/sector.ts
import { Guid } from "guid-typescript";
import { ServiceOwner } from "../service.owner/service.owner";
import { Service } from "../service/service";
import { CategoryService } from "../category.service/category.service";

export interface Sector {
    Id: Guid;
    TypeSector:string;
    IsAccept:boolean;
    Description: string;
    IsDeleted: boolean ;
    ServiceOwners: ServiceOwner[];
    Services: Service[];
    CategoryServices: CategoryService[];
}